Mage.Tests/src/test/java/org/mage/test/cards/enchantments/AnimateDeadTest.java

```diff
@@ -202,4 +202,19 @@ public void testAnimateAndDragonlordAtarkaWithNoTargets() {
         assertPermanentCount(playerA, "Animate Dead", 1);
         assertPermanentCount(playerA, "Dragonlord Atarka", 1);
     }
+
```

Comment:
Done, but I used two vanilla creatures for the test because I didn't want to deal with stacking a bunch of other triggers in addition to the Animate Dead ones.